src/Liquidator.sol

```diff
@@ -448,23 +440,37 @@ contract Liquidator is Owned, ILiquidator {
         uint256 accountValue = IAccount(account).getAccountValue(IAccount(account).baseCurrency());
         if (accountValue != 0) revert Liquidator_AccountValueIsNotZero();
 
+        _endAuction(
+            account,
+            auctionInformation_.originalOwner,
+            auctionInformation_.initiator,
+            auctionInformation_.creditor,
+            auctionInformation_.startDebt
+        );
+    }
+
+    /**
+     * @notice Ends an auction, settles the liquidation and transfers all remaining assets of the Account to the procotol owner.
+     * @param account The account to end the liquidation for.
+     */
+    function _endAuction(address account, address originalOwner, address initiator, address creditor, uint256 startDebt)
+        internal
+    {
         // Stop the auction, this will prevent any possible reentrance attacks.
         auctionInformation[account].inAuction = false;
 
-        uint256 startDebt = auctionInformation_.startDebt;
-        address owner_ = owner;
-        address creditor = auctionInformation_.creditor;
-
-        ILendingPool(creditor).settleLiquidation(
-            account, auctionInformation_.originalOwner, startDebt, auctionInformation_.initiator, owner_, 0
-        );
+        ILendingPool(creditor).settleLiquidation(account, originalOwner, startDebt, initiator, msg.sender, 0);
 
-        // Transfer all the left-over assets to the 'to' address
-        IAccount(account).auctionBoughtIn(owner_);
+        // Transfer all the left-over assets to the protocol owner.
+        IAccount(account).auctionBoughtIn(owner);
```

Comment:
```suggestion
        IAccount(account).auctionBoughtIn(owner_);
```

Saves gas, otherwise it's an extra storage read while it's already in memory